sla_per_use_system_test.py

```diff
@@ -0,0 +1,394 @@
+                return None
+            self.assertEqual(username, users_names[i], msg='Expected that stress was run with user "{}" but it was "{}"'
+                             .format(users_names[i], username))
+            results[username] = int(op_rate)
+
+        end = int(time.time())
+        scylla_load = self.prometheus_stats.get_scylla_reactor_utilization(start_time=start, end_time=end)
+        if scylla_load < waited_cpu_min:
+            self.log.warning('Load isn\'t high enough. The test results may be not correct')
+
+        return results
+
+    def clean_auth(self, entities_list_of_dict):
+        for entity in entities_list_of_dict:
+            sl = entity.get('service_level')
+            role = entity.get('role')
+            user = entity.get('user')
+            if user:
+                user.drop()
+            if role:
+                role.drop()
+            if sl:
+                sl.drop()
+
+        self.backgroud_task = None
+
+    # TODO: decide if need that
+    # def create_background_load(self, duration='120m', timeout=900, cpu_minimum=99, sleep_time_sec=120,
+    #                            rows_increase=500000):
+    #     stress_cmd = self.STRESS_WRITE_DURATION_CMD.format(duration=duration, user=self.background_user[0],
+    #                                                        password=self.background_user[0],
+    #                                                        threads=250, pop='%d..%d' % (self.n, self.n+rows_increase))
+    #     self.backgroud_task = self.run_stress_thread(stress_cmd=stress_cmd)
+    #
+    #     return wait.wait_for(func=self.wait_for_load_reached, step=sleep_time_sec, timeout=timeout,
+    #                          cpu_minimum=cpu_minimum, sleep_time_sec=sleep_time_sec)
+    #
+    # def wait_for_load_reached(self, cpu_minimum=99, sleep_time_sec=120):
+    #     end = int(time.time())
+    #     start = int(time.time())-sleep_time_sec
+    #     cpu = self.prometheus_stats.get_scylla_reactor_utilization(start_time=start, end_time=end)
+    #     if cpu and cpu > cpu_minimum:
+    #         self.log.debug("CPU load reached {}%".format(cpu_minimum))
+    #         return cpu > cpu_minimum
+    #     return False
+
+    def test_run_all(self):
+        """
+        Start all tests one by one
+        """
+        session = self.prepare_schema()
+
+        # Sleep after writes and before read load
+        time.sleep(300)
+        results = {'sla_per_user_basic': self.sla_per_user_basic(session=session)}
+
+        # Sleep between tests
+        time.sleep(300)
+        results['throughput_latency_load'] = self.throughput_latency_load(session=session)
+
+        failed_tests = ['Test {}. Error: {}'.format(test, result) for test, result in results.iteritems() if result]
+        self.assertFalse(failed_tests,
+                         msg='Failed tests: {}'.format('\n'.join(test for test in failed_tests)))
+
+    def sla_per_user_basic(self, session):
```

Comment:
add a test_ to the name

Reply:
Also change the name to describe better the scenario, maybe use the names we used in the presentation.